CodeGen/src/AssemblyBuilderA64.cpp

```diff
@@ -1226,6 +1265,17 @@ void AssemblyBuilderA64::placeER(const char* name, RegisterA64 dst, RegisterA64
     commit();
 }
 
+void AssemblyBuilderA64::placeVR(const char* name, RegisterA64 dst, RegisterA64 src1, RegisterA64 src2, uint16_t op, uint8_t op2)
```

Comment:
placeR3 with `sizes` already covers this form.

Reply:
Ah, I see, it's for logging.

Reply:
No, please leave the .4s.
But it seems we can remove the `sizes` overload later and keep VR.